packages/dom/src/dom.js

```diff
@@ -407,20 +407,6 @@ export function placeCaretAtVerticalEdge( container, isReverse, rect, mayUseScro
 		return;
 	}
 
-	// Check if the closest text node is actually further away.
-	// If so, attempt to get the range again with the y position adjusted to get the right offset.
```

Comment:
I have no idea what this code is for. Apparently I wrote it! :) Since we're moving from a `br` element to a text element for an empty rich text, this calculation makes `should navigate contenteditable with padding` fail.

Since I don't understand what this is for, and there are no failing e2e tests when removing the code, I think it's safe to remove. If we end up needing it again, we should comment it better and add e2e tests.

Reply:
@aduth This appeared to be the culprit of the e2e test failure.